.eslintrc

```diff
@@ -20,6 +20,9 @@ rules:
   one-var-declaration-per-line: off
   no-console: [1, { "allow": ["warn", "error"] }]
   no-param-reassign: [1, { "props": false }]
+  no-unused-expressions: ['error', {"allowTernary": true }]
+  no-restricted-syntax: ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'] # allow ForOfStatement
+  array-callback-return: off # we commonly mutate in place in array map statements
```

Comment:
I really liked this actually as this is a simple fix of `map` -> `forEach` for proper JS. I'd recommend to keep this rule.

Reply:
fine with me - we can explicitly return `undefined` in these cases

Reply:
I was meaning that if you're not returning something to build a new array you shouldn't be using `map`. You should be using `forEach` instead. This is just a simple word swap.